src/views/virtualmachinesinstance/details/tabs/details/components/Details/SSHAccess/SSHAccess.tsx

```diff
@@ -20,34 +27,47 @@ type SSHAccessProps = {
 const SSHDetails: React.FC<SSHAccessProps> = ({ vmi, vm, sshService, sshServiceLoaded }) => {
```

Comment:
The file name is `SSHAccess.tsx` and the props type is `SSHAccessProps`. Why is the component name `SSHDetails`? I see there is already an `SSHAccess` component, so I assume the file name and props types should be renamed to match the component name?

Reply:
Yes okay. I'll change the file name to `SSHDetails`